debug.sh

```diff
@@ -1,6 +1,6 @@
 rm ./builder/target/bootloader.img
 rm -r target/x86_64-bootloader
-cargo xbuild --target ./x86_64-bootloader.json
-cd ./builder && cargo run ../target/x86_64-bootloader/debug/robustloader ../kernel.elf # TODO: remove this example non existent kernel.elf
+cargo xbuild --target ./x86_64-bootloader.json --release
+cd ./builder && cargo run ../target/x86_64-bootloader/release/robustloader ../kernel.elf # TODO: remove this example non existent kernel.elf
 cd ../ && qemu-system-x86_64 -s ./builder/target/bootloader.img &
-gdb -ex "target remote localhost:1234" -ex "symbol-file ../target/x86_64-bootloader/debug/robustloader"
+gdb -ex "target remote localhost:1234" -ex "symbol-file ../target/x86_64-bootloader/release/robustloader"
```

Comment:
Why is debug.sh compiling with release?

Reply:
runs much faster with release, with no real difference